### PR TITLE
Add support for setting job_rank job option

### PR DESCRIPTION
**Pull request by lightster**
Issue #66

### CODE REVIEW

**Comment by lightster** [comment]
Hi @Twenty7,

Here is another small change. The change adds support for providing `job_rank` when queueing a job. Previously `job_rank` was always defaulted to 5.

Thanks,

Lightster

File: src/Hodor/JobQueue/JobOptions/Validator.php [hunk 1]
```diff
@@ -85,4 +86,19 @@ private function validateRunAfter(array $options)
...
         throw new Exception('\'run_after\' must be an instance of \DateTime');
     }
+
+    /**
+     * @param array $options
+     * @throws Exception
+     */
+    private function validateJobRank(array $options)
+    {
+        if (is_int($options['job_rank'])
```
Comment (Twenty7) [inline src/Hodor/JobQueue/JobOptions/Validator.php:96]: job_rank will always exist in the array at this point, correct?

**Comment by Twenty7** [comment]
Looks good, only one question.

File: src/Hodor/JobQueue/JobOptions/Validator.php (hunk 1 above)
Comment (lightster) [inline src/Hodor/JobQueue/JobOptions/Validator.php:96]: Correct. The `validateJobOptions` method loops through each of the job options passed to the job and calls the `validateJobRank` method only if `job_rank` is in the job options.

**Comment by lightster** [comment]
Thanks for the review!

**Pull request merged**